pcdsdevices/lic_2d_tmo.py

```diff
@@ -0,0 +1,46 @@
+"""
+Module for the two dimention laser coupling in TMO.
+"""
+from lightpath import LightpathState
+from ophyd.device import Component as Cpt
+
+from .device import GroupDevice
+from .epics_motor import BeckhoffAxisEPS
+from .interface import BaseInterface, LightpathMixin
+
+
+class TMOLaserCouplingTwoDimention(BaseInterface, GroupDevice, LightpathMixin):
+    """
+
+    TMO two dimention laser coupling LI2K4 class.
```

Comment:
Yes, Tyler. It should be a state mover screen but because the special request of this state mover LI2K4, PLC level code has not completed. State mover will not work right now. Scientists just want to move this motor from IN position to OUT. So they agree to only have two motor interfaces for the coming beam tomorrow so that they can move motor from IN to OUT.  This is not a complete GUI deployment, just a temperature one for the SC commission. After I release this, I will come back to release a real state mover (while PLC work is done).

Reply:
Scientists will not need this motor fully function until Oct. Currently they just want to see the X,Y motor screen to move from IN to OUT(laser scientists use it as it is IN position, while SC commission does not need this at all)

Reply:
@tongju12 I see, OK, that makes sense. 
Other than some typos and potential class name update, the code and generated screen look OK to me.

Reply:
Tyler, I just pushed the new code with fixing the typo errors. Thanks!

Reply:
@tongju12 I had one previous question about the class name that has not been answered. Should this be called `TMOLaserInCouplingTwoDimension`, rather than `TMOLaserCouplingTwoDimension` (note the `In`, since the common component here is laser in-coupling).

Reply:
I did not see that question. Thank you Tyler, this is a great comment. Yes, I change to laserIn, because it is coming in to the IP1. Thanks a lot! I update it already